scripts/day2.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+function deploy_day2_service() {
```

Comment:
let's rename deploy_ocp_service and the file should be ocp.sh

Reply:
done